File: __tests__/__fixtures__/output/php/guzzle/jsonObj-null-value.php
<?php

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'https://httpbin.org/anything', [
  'body' => '{"foo":null}',
  'headers' => [
    'content-type' => 'application/json',
  ],
]);

echo $response->getBody();

File: __tests__/fixtures/output/csharp/restsharp/jsonObj-null-value.cs
var client = new RestClient("https://httpbin.org/anything");
var request = new RestRequest(Method.POST);
request.AddHeader("content-type", "application/json");
request.AddParameter("application/json", "{\"foo\":null}", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);
